src/io/tree.cpp

```diff
@@ -346,9 +342,10 @@ std::string Tree::NumericalDecisionIfElse(int node) const {
   std::stringstream str_buf;
   uint8_t missing_type = GetMissingType(decision_type_[node]);
   bool default_left = GetDecisionType(decision_type_[node], kDefaultLeftMask);
-  if (missing_type == 0 || (missing_type == 1 && default_left && kZeroThreshold < threshold_[node])) {
+  if (missing_type == MissingType::None 
+  || (missing_type == MissingType::Zero && default_left && kZeroThreshold < threshold_[node])) {
```

Comment:
@AlbertoEAF 
Please fix indent here:
```suggestion
  if (missing_type == MissingType::None 
      || (missing_type == MissingType::Zero && default_left && kZeroThreshold < threshold_[node])) {
```